src/controllers/update/due.diligence.controller.ts

```diff
@@ -0,0 +1,60 @@
+import { NextFunction, Request, Response } from "express";
+import { Session } from "@companieshouse/node-session-handler";
+
+import * as config from "../../config";
```

Comment:
Just import what you need here
UPDATE_DUE_DILIGENCE_PAGE
WHO_IS_MAKING_UPDATE_URL
UPDATE_DUE_DILIGENCE_PAGE
OVERSEAS_ENTITY_REVIEW_URL